src/Test/TestAIBehavior.java

```diff
@@ -62,17 +69,19 @@ public boolean runTest()
   {
     boolean testPassed = true;
 
+    testPassed &= validate(testWalkInLine(Muriel.info), "  "+Muriel.info.getName()+" failed line walking test.");
```

Comment:
Does Wally not pass this one? Or does it just assign the closer infantry and the other one is jobless? Hm; should maybe modify the test to put two cappable properties on the right.

Reply:
Yeah, he started failing it at some point in the process and I decided I didn't care :P
That's not a bad idea, though.